spec/features/idv/doc_auth/send_link_step_spec.rb

```diff
@@ -186,4 +187,39 @@
     document_capture_session.reload
     expect(document_capture_session).to have_attributes(requested_at: a_kind_of(Time))
   end
+
+  context 'when user is from Puerto Rico' do
+    let(:area_code) {}
+    let(:phone) { "+1 #{area_code}-555-1212" }
+    let(:formatted_phone) { "(#{area_code}) 555-1212" }
+    let(:user) do
+      create(
+        :user,
+        :signed_up,
+        with: { phone: phone },
+        password: Features::SessionHelper::VALID_PASSWORD,
+      )
+    end
+
+    shared_examples 'they are a person with a real telephone' do
+      it "defaults phone to user's 2fa phone number", :js do
+        field = page.find_field(t('two_factor_authentication.phone_label'))
+        expect(field.value).to eq(formatted_phone)
+      end
+      it "allows submitting to user's 2fa phone number", :js do
+        click_idv_continue
+        expect(page).to have_current_path(idv_doc_auth_link_sent_step)
+      end
+    end
+
+    context 'with area code 787' do
+      let(:area_code) { '787' }
+      it_behaves_like 'they are a person with a real telephone'
+    end
+
+    context 'with area code 939' do
+      let(:area_code) { '939' }
+      it_behaves_like 'they are a person with a real telephone'
+    end
```

Comment:
With feature specs being very slow, is it important for us to test all of the area codes for Puerto Rico, or could we suffice with just one?

Reply:
Yeah, we can limit to one area code. Fixed in 7cac548